modules/ckanext-ytp_main/ckanext/ytp/templates/package/snippets/categories.html

```diff
@@ -15,11 +17,11 @@
       />
       {% if category.get('title_translated', {}).get(lang) %}
       <h5 class="categories__list__item__title">
-        {{ category.get('title_translated', {}).get(lang) }}
+        <a href="{{h.url_for('group_read', id=category.get('name', {}))}}">{{ category.get('title_translated', {}).get(lang) }}</a>
```

Comment:
Could be rewritten with link_for snippet like in https://github.com/vrk-kpa/opendata-ckan/pull/121/commits/1b0f40caa56d0db9208e927087cb8b87bc609730

Reply:
Yeah I did notice that this issue is a duplicate of https://jira.dvv.fi/browse/AV-1672 (or I guess that's duplicate of this since this one is older) right before my last week's vacation but only realized yesterday AFTER the vacation that I completely forgot to mention it... 😬 

In https://github.com/vrk-kpa/opendata-ckan/pull/121 I first made essentially the same fix, then I read that issue again and realized it said it should work similar to keywords so made another change. Which I now reverted after discussing with Jari how it should work.